sources/api/apiserver/src/server/mod.rs

```diff
@@ -703,18 +703,19 @@ async fn list_ephemeral_storage_dirs(
 ) -> Result<HttpResponse> {
     let os_info = controller::get_os_info()?;
 
-    let allowed = ephemeral_storage::allowed_bind_dirs(&os_info.variant_id);
+    let (allowed_exact, _allowed_prefixes, _disallowed_contains) =
```

Comment:
nit: it's fine to just ignore and not use these returns:
```suggestion
    let (allowed_exact, _, _) =
```

Reply:
Thanks, done this in most latest commit.